www/views/hardware.html

```diff
@@ -1662,6 +1662,15 @@ <h2><span data-i18n="Hardware">Hardware</span>:</h2>
                         <br>
                         <table class="display" id="hardwareparamsgoodweweb" border="0" cellpadding="0" cellspacing="20">
                                 <tr>
+					<td align="right" style="width:110px"><label for="comboserverselect"><span data-i18n="Select a Server">Select a Server</span>:</label></td>
+					<td><select id="comboserverselect" style="width:200px" class="combobox ui-corner-all">
+						<option value="0">Global</option>
+						<option value="1">Oceania</option>
+						<option value="2">Europe</option>
+						</select>
+					</td>
+				</tr>
+				<tr>
 					<td align="right" style="width:110px"><label><span data-i18n="username">Goodwe-power Username</span>:</label></td>
```

Comment:
Could you change data-i18n="username" to use "Username" (capital U), this one is already in use and translated.
Also >Goodwe-power Username rename to 'Username' only, like the data-i18n, on better, you can remove that

Reply:
Best to open 'a' language file from www/i18n folder, and search what you can use